src/app/api/team-member/[memberId]/route.js

```diff
@@ -0,0 +1,18 @@
+import axios from "axios";
+import { NextResponse } from "next/server";
+
+const getATeamMember = async (req, res) => {
```

Comment:
```suggestion
const getATeamMember = async (req) => {
```

Next 13.4 onwards, the API routes' handler doesn't receive a response object

```javascript
const someRandomHandler = async (req) => {
...
} 
```